executors/dart_web/out/executor.js

```diff
@@ -0,0 +1,191 @@
+/* Main execution program for Data Driven Test of ICU / Unicode / CLDR
```

Comment:
This is just a copy of the node executor, I don't know what the smartest way is to not duplicate the file. Copying at runtime seems ugly...

Reply:
OK for now, but copy will be a problem when the node version gets out of sync with Dart implementation.